test/gtest/ucs/test_event_set.cc

```diff
@@ -128,11 +135,14 @@ UCS_TEST_F(test_event_set, ucs_event_set_write_thread) {
     EXPECT_EQ(UCS_OK, status);
     EXPECT_TRUE(event_set != NULL);
 
-    status = ucs_event_set_add(event_set, pipefd[1], UCS_EVENT_SET_EVWRITE);
+    status = ucs_event_set_add(event_set, pipefd[1], UCS_EVENT_SET_EVWRITE,
+                               (void *)&pipefd[1]);
     EXPECT_EQ(UCS_OK, status);
 
-    status = ucs_event_set_wait(event_set, 50, event_set_func2, NULL);
-    EXPECT_EQ(UCS_OK, status);
+    nread = ucs_event_set_wait(event_set, MAX_EVENT_SET_SIZE, 50,
+                               event_set_func2, NULL, &status);
+    EXPECT_EQ(1, nread);
+//    EXPECT_EQ(NULL, &status);
```

Comment:
pls, remove

Reply:
I checked UCS_STATUS instead of NULL check